config/configtls/README.md

```diff
@@ -18,14 +18,18 @@ As a result, the following parameters are also required:
 
 - `cert_file`: Path to the TLS cert to use for TLS required connections. Should
   only be used if `insecure` is set to false.
+  - `cert_pem`: Alternative to `cert_file`. Provide the certificate contents as a string instead of a filepath.
+
 - `key_file`: Path to the TLS key to use for TLS required connections. Should
   only be used if `insecure` is set to false.
```

Comment:
@bogdandrutu @jpkrohling should we deprecate the `*_file` options in favor of the `*_pem` options, given that people can do: `cert_pem: ${file:/path/to/cert}` now?

Reply:
That's a great idea

Reply:
Tracking issue?

Reply:
#7691